pkgs/tools/audio/beets/plugins/filetote.nix

```diff
@@ -0,0 +1,90 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3Packages,
+  beets,
+  beetsPackages,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "beets-filetote";
+  version = "1.0.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "gtronset";
+    repo = "beets-filetote";
+    rev = "v${version}";
+    hash = "sha256-q+CWn8p7/16Z6mAzOfh0dqg96CO7X1u/jHPuhK61bVw=";
+  };
+
+  postPatch = ''
+    sed -i -e '/install_requires/,/\]/{/beets/d}' setup.py
+    sed -i -e '/namespace_packages/d' setup.py
+    printf 'from pkgutil import extend_path\n__path__ = extend_path(__path__, __name__)\n' >beetsplug/__init__.py
```

Comment:
I'm not sure what this is for? The file [exists upstream](https://github.com/gtronset/beets-filetote/blob/v1.0.0/beetsplug/__init__.py) and has the same content, except with `__version__ = "1.0.0"` added. Is that a problem?